packages/subfooter/src/SubFooter.js

```diff
@@ -1,21 +1,23 @@
 import React from 'react';
 import StandardSubFooter from './Blocks/StandardSubFooter';
 
-import './Subfooter.scss';
+import './SubFooter.scss';
 
 type Props = {
   /** Get standard subfooter such as for type 'antwerpen' */
   type?: string,
   children?: any,
+  /** Qa id */
```

Comment:
voor mij was het wel echt niet nodig da je die (niet veel zeggende) comment in elke file toevoegd :p (niet nodig om dit aan te passen)

Reply:
Noted. 😉